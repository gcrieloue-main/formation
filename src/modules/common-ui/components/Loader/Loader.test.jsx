import React from 'react';
import TestRenderer from 'react-test-renderer';

import { Loader } from '.';

it('renders correctly a layout component', () => {
  const testRenderer = TestRenderer.create(<Loader />);
  expect(testRenderer.toJSON()).toMatchSnapshot();
});
