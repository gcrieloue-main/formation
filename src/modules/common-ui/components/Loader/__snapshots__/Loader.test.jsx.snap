// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly a layout component 1`] = `
<div
  className="loaderContainer"
>
  <div
    className="ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
